### PR TITLE
Added vendor extension for local token validation.

### PR DESCRIPTION
**Pull request by MaicoTimmerman**
Implements a method to locally validate the Bearer token passed to endpoints. See #427.

### CODE REVIEW

File: connexion/decorators/security.py [hunk 1]
```diff
@@ -17,14 +18,27 @@
...
+def get_tokeninfo_func(security_definition):
+    """
+    :type security_definition: dict
+    :rtype: function
+
+    >>> get_tokeninfo_url({'x-tokenInfoFunc': 'foo.bar>'})
+    '<function foo.bar>'
+    """
+    token_info_func = (security_definition.get("x-tokenInfoFunc") or
+                       os.environ.get('TOKENINFO_FUNC'))
```
Comment (hjacobs) [inline connexion/decorators/security.py]: passing an (internal) function name via environment variable does not make sense I think

Comment (MaicoTimmerman) [reply connexion/decorators/security.py]: This function name is the function in the users application, not in the library. So I think we should offer the same methods for the tokenInfoFunc as the tokenInfoUrl.

What is the reason you'd not want to include this?

Comment (hjacobs) [reply connexion/decorators/security.py]: @MaicoTimmerman I can just not a find a good reason for it: the app and the swagger file should both be static (i.e. compile-time), environment variables are for runtime configuration and I don't see a good argument why you would want to switch the function at runtime (startup). We could leave it in, but I stumbled over it and it immediately raised the question "why? what's the benefit?".

Comment (hjacobs) [reply connexion/decorators/security.py]: On the other hand I could imagine having a "noop" token validation function for local testing, so maybe your `TOKENINFO_FUNC` makes sense to overwrite it with empty value "" (no token validation).

Comment (MaicoTimmerman) [reply connexion/decorators/security.py]: I'd personally prefer to keep it like it is currently, so can we proceed to merge the PR?

**Pull request merged**